src/customer-service/Data/tables.sql

```diff
@@ -68,4 +33,4 @@ CREATE TABLE "Incident"
 	FOREIGN KEY ("IncidentTypeId") REFERENCES "IncidentType" ("IncidentTypeId"),
 	FOREIGN KEY ("OrderId") REFERENCES "Order" ("OrderId"),
```

Comment:
This table doesn't exist in the current context, need to drop this FOREIGN KEY statement